index.html

```diff
@@ -69,47 +69,36 @@ <h1>CODEVID-19</h1>
     <div class="inner">
       <h2>Rules</h2>
       <p>
-        The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are
-        awarded until April 30th, when projects can submit their projects for
-        the final prize consideration. Final prizes are awarded shortly after
-        the judging period concludes.
+        The CODEVID-19 Hackathon is comprises of weekly and final prizes. Weekly
+        prizes are awarded until the week of April 30th: that's when
+        participants can submit their projects for final prize consideration.
       </p>
 
-      <p>Projects compete in <b>four problem areas</b>:</p>
+      <p>Projects compete in <strong>four problem areas</strong>:</p>
```

Comment:
Not part of this PR. What's the reasoning for this change?

Reply:
I can revert.  Originally it was for consistency and personal preference.  I thought `<b>` tags were falling out of favour and I personally use `<strong>` over them in my own projects.  That said, both tags are valid according to our SASS.